posthog/settings.py

```diff
@@ -331,9 +335,10 @@ def get_bool_from_env(name: str, default_value: bool) -> bool:
         "BACKEND": "django.core.cache.backends.locmem.LocMemCache",
     }
 
-
 if DEBUG and not TEST:
-    print("🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰")
-    print("️🧰 🔧 Running PostHog in __development mode__! DEBUG=1 🔧 🧰")
-    print("️🧰 ⚠️ Please update your config if this is a live site ⚠️ 🧰")
-    print("🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰🧰")
+    print_warning(
+        (
+            "️Environment variable DEBUG is set - PostHog is running in development mode!",
+            "Absolutely unset DEBUG if this is supposed to be a production environment!",
```

Comment:
This isn't quite correct English. Maybe `Please unset DEBUG if this is a production environment`